referrer-policy/get_failing_tests.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import json
+
+data = json.load(open("attr-referrer-failing.json"))
+
+prefix = "http://web-platform.test:8000"
```

Comment:
This only works locally. Is this file intended to do anything when requested over the web? If not, I'd suggest moving it to tools, and adding some comments about its intended use case.